Source/WebCore/platform/graphics/displaylists/DisplayList.h

```diff
@@ -44,6 +44,10 @@ class DisplayList {
     WTF_MAKE_NONCOPYABLE(DisplayList);
 public:
     DisplayList() = default;
+    DisplayList(const ApplyItemOptions& options)
+        : m_options(options)
```

Comment:
Which part of the code will pass these `ApplyItemOptions`? I will be great if this happens in this patch even if you pass the default option for SKIA for now.

Reply:
Great idea -- thanks @shallawa. PR #34502 contains the patch that makes use of the new `ApplyItemOptions` but I like your idea much more, to make it visible from the beginning where it's intended to be used.

Will adapt the PR and come up with a new version soon!